### PR TITLE
Move monorepo to NPM@8 workspaces

### PR DESCRIPTION
**Pull request by andrey-skl**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ### Environment setup
 
 1. (macOS only) Install Xcode Command Line Tools: `xcode-select --install`
-2. Install Node.js
-4. Bootstrap the packages: `npm run bootstrap`
-5. (Optional, macOS and Linux) Install the [status bar indicator](https://github.com/roman01la/anybar-webpack#known-apps) app to receive webpack build notifications.
+2. Install Node.js@16 (with NPM@8). We suggest using NVM for Node version management
```
Comment (Hypnosphi) [inline CONTRIBUTING.md:4]: I find `n` more convenient

File: src/shortcuts/shortcuts-hoc.tsx [hunk 2]
```diff
@@ -35,6 +35,8 @@ export default function shortcutsHOC<P>(ComposedComponent: ComponentType<P> | st
           options={rgShortcutsOptions}
           disabled={rgShortcutsOptions.disabled}
         >
+          {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+          {/* @ts-ignore */}
```
Comment (Hypnosphi) [inline src/shortcuts/shortcuts-hoc.tsx:39]: What was the TS error here?

**Review by Hypnosphi — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrey-skl) [reply CONTRIBUTING.md:4]: @Hypnosphi yep that makes sense. Let's suggest to use any of these

File: src/shortcuts/shortcuts-hoc.tsx (hunk 2 above)
Comment (Hypnosphi) [reply src/shortcuts/shortcuts-hoc.tsx:39]: It's a new error in TS 4.7 https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#stricter-spread-checks-in-jsx

Comment (Hypnosphi) [reply src/shortcuts/shortcuts-hoc.tsx:39]: Just pin the TS version to `~4.6`, I'll update it later

**Pull request merged**